### PR TITLE
Release note for 0.44.0 release

### PR DESCRIPTION
**Pull request by Sreekala-Gopakumar**
Created the release note for 0.44.0 release with the available issue details. Updated 0.41.0 release note based on a change documented in 0.44.0 release

[skip ci]
Signed-off-by: Sreekala Gopakumar sreekala.gopakumar@ibm.com

### CODE REVIEW

**Comment by Sreekala-Gopakumar** [comment]
@pshipton - I have created a draft release note for 0.44.0 release. I will update this as and when the documentation of any other issues are completed. Thanks!

File: doc/release-notes/0.41/0.41.md [hunk 1]
```diff
@@ -68,6 +68,19 @@ The following table covers notable changes in v0.41.0. Further information about
 <td valign="top">In the earlier releases, if the <tt>-Xshareclasses:readonly</tt> option and the JITServer AOT cache feature were both enabled at the same time at a JITServer client, the client could not use the JITServer AOT cache because the cache for storing data that the JITServer AOT cache needed was read-only.
 Now, with the change in behavior of the <tt>-Xshareclasses:readonly</tt> option, the shared class cache startup creates a temporary new (writable) top layer that the JITServer AOT cache can use to store data that it needs to function.</td>
 </tr>
+
+<tr>
+<td valign="top"><a href="https://github.com/eclipse-openj9/openj9/issues/18401">#18401</a>, a href="https://github.com/eclipse-openj9/openj9/issues/18402">#18402</a></td>
```
Comment (pshipton) [inline doc/release-notes/0.41/0.41.md:73]: The second `a href` should be `<a href`

File: doc/release-notes/0.44/0.44.md [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+To resolve this issue, OpenJDK version 17 ignores these options and from 0.41.0 release onwards, OpenJDK version 11 also ignores these options. 
+
+With this release, OpenJDK version 8 too ignores the <tt>allow</tt> and <tt>disallow</tt> keywords, if specified.</td>
+</tr>
+
+<tr>
+<td valign="top"><a href="https://github.com/eclipse-openj9/openj9/issues/17990">#17990</a>, <a href="https://github.com/eclipse-openj9/openj9/issues/18845">#18845</a></td>
+<td valign="top">Display of multiple warnings on loading the same agent is now restricted on AIX systems.</td>
+<td valign="top">OpenJDK 21 and later (AIX)</td>
+<td valign="top">Earlier, on AIX systems, warnings were issued each time the agents were loaded dynamically into a running VM after startup without specifying the `-XX:+EnableDynamicAgentLoading` option. These multiple warnings were issued even if the same agents were loaded before.ß
```
Comment (pshipton) [inline doc/release-notes/0.44/0.44.md:67]: Strange character at the end `ß1`

File: doc/release-notes/0.44/0.44.md [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+
+Now, from 0.44.0 release onwards, AIX systems, like other OpenJ9 supported operating systems, can detect whether an agent was previously loaded or not. Therefore, like other platforms, on AIX systems also, the warnings are issued only once for the same agent when the `-XX:+EnableDynamicAgentLoading` option is not specified.
```
Comment (pshipton) [inline doc/release-notes/0.44/0.44.md:69]: from -> from the

File: doc/release-notes/0.44/0.44.md [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+</td>
+</tr>
+
+<tr>
+<td valign="top"><a href="https://github.com/eclipse-openj9/openj9/issues/18342">#18342</a></td>
+<td valign="top">XL C++ Runtime 16.1.0.7 or later is required for AIX OpenJ9 builds on OpenJDK 8.</td>
+<td valign="top">All versions</td>
```
Comment (pshipton) [inline doc/release-notes/0.44/0.44.md:76]: This could include `(AIX)`.

File: doc/release-notes/0.44/0.44.md [hunk 5]
```diff
@@ -0,0 +1,139 @@
...
+</thead>
+<tbody>
+
+<tr>
+<td valign="top"><a href="https://github.com/eclipse-openj9/openj9/issues/18900">#18900</a></td>
+<td valign="top">Change in behavior of the <tt>-Djava.security.manager</tt> system property for OpenJDK version 8.</td>
+<td valign="top">OpenJDK 8</td>
+<td valign="top">
+
+In OpenJDK versions before version 18, the runtime attempts to load a <tt>SecurityManager</tt> with the class name <tt>allow</tt> or <tt>disallow</tt> resulted in an error and the application was not starting.
```
Comment (pshipton) [inline doc/release-notes/0.44/0.44.md:56]: This reads funny, I think either remove "the runtime", or change "resulted" to "resulting".

**Comment by Sreekala-Gopakumar** [comment]
@pshipton - I have incorporated the changes. Thanks!

**Comment by Sreekala-Gopakumar** [comment]
@pshipton - Updated the release notes with documented issues. Please check and confirm. Thanks!

File: doc/release-notes/0.44/0.44.md [hunk 6]
```diff
@@ -0,0 +1,177 @@
...
+<td valign="top">The <tt>-XX:[+|-]ShowUnmountedThreadStacks</tt>option is added to control the inclusion of the unmounted virtual threads in a Java&trade; core file.</td>
+<td valign="top">OpenJDK 21 and later</td>
+<td valign="top">Java core file lists stacks of only those threads that are mapped to platform threads. An unmounted virtual thread is not mapped to any platform thread. Therefore, the stack of any unmounted virtual thread is not included in the Java core file and thus, the virtual thread information remains incomplete.
+
+You can use the <tt>-XX:+ShowUnmountedThreadStacks</tt> option to include all the thread data that a VM is aware of, both regular Java threads and the unmounted virtual threads, in a Java core file.</td>
+</tr>
+
+<tr>
+<td valign="top"><a href="https://github.com/eclipse-openj9/openj9/issues/19173">#19173</a></td>
+<td valign="top">Flag names that refer to the fields of <tt>J9BuildFlags</tt> is changed in the Direct Dump Reader (DDR) code</td>
```
Comment (pshipton) [inline doc/release-notes/0.44/0.44.md:91]: is changed -> are changed

File: doc/release-notes/0.44/0.44.md [hunk 7]
```diff
@@ -0,0 +1,177 @@
...
+<td valign="top">All versions</td>
+<td valign="top">User plug-ins in the DDR code might contain code that has <tt>J9BuildFlags</tt> reference to read the build flags in the system dump data. When the <tt>J9BuildFlags</tt> is generated for each build, the raw name of the build flags might be used in the plug-in code rather than the <tt>cname</tt> that appears in <tt>j9cfg.h</tt> (derived from <tt>j9cfg.h.ft1</tt>). Therefore, mapping of flag names in the plug-in code with the names that are specified in <tt>j9cfg.h</tt> was inconsistent and because of this inconsistency, some flag tests might not apply to all system dumps.
```
Comment (pshipton) [inline doc/release-notes/0.44/0.44.md:93]: reference -> references

File: doc/release-notes/0.44/0.44.md [hunk 8]
```diff
@@ -0,0 +1,177 @@
...
+
+To reduce complexity and improve the uniformity of how the DDR code uses the flags, you must use the names that are specified in <tt>j9cfg.h</tt> as flag names in the plug-in code.
+</td>
+</tr>
+
+<tr>
+<td valign="top"><a href="https://github.com/eclipse-openj9/openj9/issues/19045">#19045, <a href="https://github.com/eclipse-openj9/openj9/issues/19069">#19069,</a></td>
+<td valign="top">VMID query in the <tt>jcmd</tt> tool enhanced</td>
+<td valign="top">All versions</td>
+<td valign="top">Earlier in OpenJ9, when sending a <tt>jcmd</tt> command to a VM, you had to run <tt>jcmd -l</tt> to retrieve all the pids for all the VMs found on the machine. Then, you had to use <tt>jcmd [vmid] [command]</tt> to send the command to the specific VM.
```
Comment (pshipton) [inline doc/release-notes/0.44/0.44.md:103]: all the pids for all -> the pids for all

File: doc/release-notes/0.44/0.44.md [hunk 9]
```diff
@@ -0,0 +1,177 @@
...
+The <tt>jcmd</tt> tool also now supports specifying <tt>0</tt> as a VMID to target all VMs.</td>
+</tr>
+
+<tr>
+<td valign="top"><a href="https://github.com/eclipse-openj9/openj9/issues/18720">#18720</a>, <a href="https://github.com/eclipse-openj9/openj9/issues/18996">#18996</a></td>
+<td valign="top">A new system property, <tt>-Dcom.ibm.tools.attach.fileAccessUpdateTime</tt>, is added to prevent automatic deletion of the Attach API control files in the <tt>/tmp</tt> folder.</td>
+<td valign="top">All versions (Linux&reg;)</td>
+<td valign="top">In Linux environments, by default <tt>systemd-tmpfiles</tt> automatically deletes all files and directories that are stored in the <tt>/tmp/</tt> folder that have not been changed or read within a specific time period. This deletion of files creates a problem for long running VM processes, such as Attach API, that need the control files in the <tt>/tmp/</tt> folder to operate.
+
+You can prevent the automatic deletion of the Attach API control files in the <tt>/tmp</tt> folder by setting the <tt>-Dcom.ibm.tools.attach.fileAccessUpdateTime</tt> system property.</td>
```
Comment (pshipton) [inline doc/release-notes/0.44/0.44.md:116]: This should mention it's set to 8 days by default, so you don't actually have to set the property unless the default needs to be changed.

File: doc/release-notes/0.44/0.44.md [hunk 10]
```diff
@@ -0,0 +1,177 @@
...
+</tr>
+
+<tr>
+<td valign="top">N/A </td>
+<td valign="top">CentOS Stream 8 is out of support.</td>
+<td valign="top">All versions (CentOS Stream 8) </td>
+<td valign="top">CentOS Stream 8 is EOL from May 31, 2024 onwards and is removed it the list of supported platforms.</td>
```
Comment (pshipton) [inline doc/release-notes/0.44/0.44.md:123]: it -> from

**Comment by Sreekala-Gopakumar** [comment]
@pshipton - I have incorporated the changes. Please check and confirm. Thanks!

**Review by pshipton — APPROVED** [review]

**Pull request merged**